src/config.js

```diff
@@ -2,13 +2,15 @@ const CLA = {
   KUSAMA: 0x99,
   POLKADOT: 0x90,
   POLYMATH: 0x91,
+  DOCK: 0x92,
 };
 
 // https://github.com/satoshilabs/slips/blob/master/slip-0044.md
 const SLIP0044 = {
   KUSAMA: 0x800001b2,
   POLKADOT: 0x80000162,
   POLYMATH: 0x80000001, // TODO replace with mainnet code when available
+  DOCK: 0x80000252, // TODO: PR https://github.com/satoshilabs/slips/pull/1039
```

Comment:
it seems the PR got accepted, could you now remove the TODO?

Reply:
Thanks for checking. Removed.